docs/terms/data_docs_store.md

```diff
@@ -0,0 +1,65 @@
+
+
+<UniversalMap setup='active' connect='active' create='active' validate='active'/> 
+
+## Overview
+
+### Definition
+
+A Data Docs Store is a connector to store and retrieve information pertaining to Human readable documentation generated from Great Expectations metadata detailing Expectations, Validation Results, etc.
+
+### Features and promises
+
+### Relationship to other objects
+
+Your Data Docs Store will be used behind the scenes by any Action that updates your Data Docs.  Great Expectations includes the `UpdateDataDocsAction` subclass of the `ValidationAction` class for this express purpose.  Including the `UpdateDataDocsAction` in the `action_list` of a Checkpoint will cause your Data Docs to be updated accordingly.
+
+## Use cases
+
+<SetupHeader/>
+
+When you configure your Data Docs, one of the items you will need to include is a `data_docs_store_backend` for each entry in the `data_docs_sites` key located in the `great_expectations.yml` file.  This will provide Great Expectations with the necessary information to access the Data Docs being rendered for each given site.  That process will generally be handled behind the scenes; after configuring your Data Docs sites to include a Data Docs Store, you will generally not have to directly interact with the Data Docs Store yourself.
+
+<CreateHeader/>
+
+When Profilers are run to create Expectations their results are made available through Data Docs.  Additionally, most Profiler workflows include a step where the Profiler's generated Expectation Suite is Validated against a Batch of data.  These processes use a Data Docs Store behind the scenes.
+
+<ValidateHeader/>
+
+When Checkpoints are run to Validate data, their results are generally updated into Data Docs.  Any Checkpoint that is configured to do so will use a Data Docs Store behind the scenes.  Including the `UpdateDataDocsAction` in the `action_list` of a Checkpoint will cause your Data Docs to be updated accordingly.  This update will only include the Validation Results from the Checkpoint itself and will not re-render all of your existing Data Docs to include Validation Results that were generated by a Checkpoint that did not include the `UpdateDataDocsAction` option.
+
+
+## Features
+
+### Ease of use
+
+Once your Data Docs sites are configured with a Data Docs Store backend, the Great Expectations Objects that access Data Docs will handle the rest: you won't have to directly interact with the Data Docs Store.
+
+## API basics
+
+### How to access
+
+Your Data Context and other objects that interact with Data Docs will access your Data Docs Stores behind the scenes without you needing to directly manage them yourself beyond configuring your Data Docs sites during Setup.  
+
+In the Validate Data step, including `UpdateDataDocsAction` in the `action_list` of a Checkpoint will cause your Data Docs to be updated with the Checkpoint's Validation Results; this process will use your Data Docs Stores behind the scenes.
```

Comment:
Is `data_context.build_data_docs()` worth mentioning as part of the API?

Reply:
I added it in.